src/Chainweb/Pact/TxSimulator.hs

```diff
@@ -0,0 +1,127 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Chainweb.Pact.TxSimulator
```

Comment:
If this is used only in `cwtool`, can we move the module there, in order to keep the prod code base small?